upkie/cpp/actuation/tests/BulletInterfaceTest.cpp

```diff
@@ -290,24 +290,20 @@ TEST_F(BulletInterfaceTest, ObserveImuOrientation) {
 
 TEST_F(BulletInterfaceTest, MonitorContacts) {
   Dictionary config;
-  config("bullet")("monitor")("contacts")("left_wheel_tire") = true;
-  config("bullet")("monitor")("contacts")("right_wheel_tire") = true;
+  config("bullet")("monitor")("contacts")("wheels")("include")(
+      "left_wheel_tire") = true;
+  config("bullet")("monitor")("contacts")("wheels")("include")(
+      "right_wheel_tire") = true;
   interface_->reset(config);
 
   Dictionary observation;
   interface_->cycle([](const moteus::Output& output) {});
   interface_->observe(observation);
-
+  spdlog::info(observation);
```

Comment:
```suggestion
```

Maybe you are still working on this and the PR is not ready for review yet. If so, sorry for the early jump :blush: I will wait for your signal when the PR is ready for review.

Reply:
Yep ! I will make it clean today